app/controllers/application_controller.rb

```diff
@@ -1,13 +1,16 @@
 class ApplicationController < ActionController::Base
   before_action :configure_permitted_parameters, if: :devise_controller?
 
-   protected
+  protected
 
-   def configure_permitted_parameters
-     devise_parameter_sanitizer.for(:sign_up) { |user| user.permit(:username,
-       :email, :password, :password_confirmation, :location, :age, :first_name,
-       :last_name) }
-   end
+  def configure_permitted_parameters
+    devise_parameter_sanitizer.for(:sign_up) { |user| user.permit(:username,
+      :email, :password, :password_confirmation, :location, :age, :first_name,
+      :last_name, :profile_photo) }
+    devise_parameter_sanitizer.for(:account_update) { |user| user.permit( :username,
```

Comment:
Line is too long. [84/80]<br>Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.<br>Space inside parentheses detected.